src/pages/[game]/create/index.tsx

```diff
@@ -1,8 +1,44 @@
+import { GetServerSidePropsContext } from 'next';
+
+import { GAME_T0_CATEGORY, PAGE_PATHS } from '@/constants';
+import { formatLinkToGame, getAuthCookie, isValidGameName } from '@/utils';
+
 import CreatePageContent from '@/components/createPage/CreatePageContent';
 import Layout from '@/components/layout/Layout';
 
-const CreatePage = () => {
-  return <CreatePageContent />;
+import { Category, GameNameEN, LinkName } from '@/types';
+
+export function getServerSideProps(context: GetServerSidePropsContext) {
+  const { accessToken } = getAuthCookie(context);
+  const isLoggedIn = !!accessToken;
+
+  const game = context.params?.game;
+  const isValid = isValidGameName(game as string);
+
+  if (!isLoggedIn || !isValid) {
```

Comment:
```suggestion
  if (isLoggedIn && isValid) {
```

요렇게 쓰는게 더 깔끔해 보이는군요!

Reply:
로그인이 안되어있으면 로그인 페이지로 이동시키고
유효하지 않은 주소면 리스트 페이지로 이동시키도록 수정했습니다!